Creds_AV_Update/email.html

```diff
@@ -0,0 +1,16 @@
+Good afternoon,
+
+The IT Department has just finished migrating to a new anti-virus platform which is will more accurately identify viruses and malware which may affect your systems.
```

Comment:
"...platform which is will more accurately..." That's not grammatically correct, but given it's a phishing template I'm unsure if it's intentional or not.